nidirect_workflow/src/Controller/AuditController.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
```

Comment:
No @file block comment. That'll fail PHPCS ;)

Reply:
Will add a comment after namespace as discussed